test-case/check-audio-equalizer.sh

```diff
@@ -110,7 +123,7 @@ do
     rate=$(func_pipeline_parse_value "$idx" rate)
     fmt=$(func_pipeline_parse_value "$idx" fmt)
     type=$(func_pipeline_parse_value "$idx" type)
-    eq_support=$(func_pipeline_parse_value "$idx" eq)
+    eq_support=($(func_pipeline_parse_value "$idx" eq))
```

Comment:
```suggestion
    IFS=" " read -r -a eq_support <<< "$(func_pipeline_parse_value "$idx" eq)" 
```
this way it makes shellcheck happy when I tried

Reply:
Excellent, that works!